plugins/python_3_script/icon_python_3_script/actions/run/action.py

```diff
@@ -102,13 +116,13 @@ def _execute_function_as_process(
                 execution_arguments,
                 shell=False,
                 stderr=subprocess.PIPE,
-                timeout=DEFAULT_PROCESS_TIMEOUT,
+                timeout=timeout * 60,
             )
             return extract_output_from_stdout(output.decode(DEFAULT_ENCODING), execution_id)
         except subprocess.CalledProcessError as error:
             raise PluginException(error.stderr.decode(DEFAULT_ENCODING).replace(execution_id, "")) from None
         except subprocess.TimeoutExpired:
-            raise PluginException(f"Function got timed out after {DEFAULT_PROCESS_TIMEOUT} seconds.") from None
+            raise PluginException(f"Function got timed out after {timeout} minutes.") from None
```

Comment:
```suggestion
            raise PluginException(f"Function timed out after {timeout} minutes.") from None
```
I think remove the word 'got'.